### ISSUE TITLE
[Bug] head rendering issue with MCA and "Regen" mod

### ISSUE THREAD

**Issue by AlienXtream** [tr7zw/FirstPersonModel#322]
three way issue with these mods where the villager model for the player renders the head in first person. likely on those two mods end but linking here for convenience as it is relevent to this mod too. doubt you need to do anything about this @tr7zw 

https://github.com/Luke100000/minecraft-comes-alive/issues/469

**Comment by tr7zw** [comment]
I have seen issues with MCA's "player model" mode causing issues, compared to the "vanilla model" mode. Probably related.

**Comment by AlienXtream** [comment]
fair enough. TBF i think they have to do what they do for their skins system to work so i understand the fact it causes some issues as a result. they fixed this one once before on their end (doing a check for FPM and adjusting their rendering to account for it) and indeed with just MCA and FPM the fix does work. similarly, with just regen and FPM its fine. its only when all 3 are together that it resurfaces. im assuming that something Suffs is doing to do skin rendering is changing some part of the player model data which is in turn negating the fix that MCA does to support FPM. yeh... 3-way mod bugs are fun, arent they XD

**Comment by AlienXtream** [comment]
fortunately its not gamebreaking as disabling MCA's rendering stuff does avoid this issue. naturally though that does mean you cant use MCA skins. again, not game breaking. just a but limiting for RP stuff
